recipes/log.c/all/CMakeLists.txt

```
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(log.c 
+	    VERSION 0.1.0 
+	    DESCRIPTION "A simple logging library implemented in C99" 
+	    HOMEPAGE_URL "https://github.com/rxi/log.c"
+	    LANGUAGES C
+	)
+
+# Init Conan stuff
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+# Set paths
+include(GNUInstallDirs)
+set(SRC_FOLDER "source_subfolder")
+
+# Library files SHARED or STATIC set by Conan with BUILD_SHARED_LIBS
+add_library(${PROJECT_NAME} ${SRC_FOLDER}/src/log.c ${SRC_FOLDER}/src/log.h)
+
+# Library info
+set_target_properties(${PROJECT_NAME} PROPERTIES
+    VERSION ${PROJECT_VERSION})
+
+# Conan options
+if (${LOG_USE_COLOR})
```

No `${..}` needed in an if.
```suggestion
if (LOG_USE_COLOR)
```